woodwork/table_accessor.py

```diff
@@ -145,6 +146,27 @@ def __getattr__(self, attr):
         else:
             raise AttributeError(f"Woodwork has no attribute '{attr}'")
 
+    def __getitem__(self, key):
+        if isinstance(key, list):
+            index = pd.Index(key)
+            columns = self._dataframe.columns
+            diff = index.difference(columns)
+            in_columns = diff.isin(columns).all()
+
+            if not in_columns:
+                raise KeyError(f"Column(s) '{list(diff)}' not found in DataTable")
```

Comment:
Thinking maybe it would be good to be consistent with other places where we do checks like this and use sets to identify differences between the supplied values and the columns in the DataFrame. 

Existing code for example: https://github.com/alteryx/woodwork/blob/f18395909c57061415242b8b65cabed99477bd5c/woodwork/table_accessor.py#L736-L741

Not sure which is most efficient...

Reply:
Another note, if we do use sets, we should probably sort the list of columns that gets displayed in the error message to make sure they are always in the same order.

Reply:
✅   0901d4d